app/routes/notifications.js

```diff
@@ -1,7 +1,22 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class NotificationsRoute extends Route {
-  async model() {
+  @service router;
+
+  queryParams = {
+    dev: {
+      refreshModel: false,
+    },
+  };
+
+  beforeModel(transition) {
+    if (transition?.to?.queryParams?.dev !== 'true') {
+      this.router.transitionTo('');
+    }
+  }
+
+  async model(params) {
```

Comment:
Can we please remove the argument `params` as it is not being used and the build is failing due to this.